fhirstore/fhirstore.py

```diff
@@ -113,6 +115,8 @@ def create(self, resource, bypass_document_validation=False):
                 resource, bypass_document_validation=bypass_document_validation
             )
             return {**resource, "_id": res.inserted_id}
+        except DuplicateKeyError as e:
```

Comment:
I guess those 2 lines are not needed because DuplicateKeyError doesn't subclass  WriteError right?

Reply:
I think it does, WriteError is the base exception type for error that happen during writing.
Also, I don't want to do the same thing when the key is already indexed and when the validation fails.
Do you think there is a better way to write this?